src/sentry/static/sentry/app/components/timeSince.jsx

```diff
@@ -73,7 +76,7 @@ class TimeSince extends React.PureComponent {
     return (
       <time
         dateTime={date.toISOString()}
-        title={moment.tz(date, options.timezone).format(format)}
+        title={this.state.momentFunc(date, options.timezone).format(format)}
```

Comment:
It'd be nice if, instead of re-doing this pattern in every translation aware element, we had a generic `moment` proxy/component that replaced itself. Otherwise I'm worried people will forget to do this moving forward.